auth_jwt_test.go

```diff
@@ -1195,3 +1197,28 @@ func TestCheckTokenString(t *testing.T) {
 	assert.Error(t, err)
 	assert.Equal(t, MapClaims{}, ExtractClaimsFromToken(nil))
 }
+
+func TestLogout(t *testing.T) {
+	cookieName := "jwt"
+	cookieDomain := "example.com"
+	// the middleware to test
+	authMiddleware, _ := New(&GinJWTMiddleware{
+		Realm:         "test zone",
+		Key:           key,
+		Timeout:       time.Hour,
+		Authenticator: defaultAuthenticator,
+		SendCookie:    true,
+		CookieName:    cookieName,
+		CookieDomain:  cookieDomain,
+	})
+
+	handler := ginHandler(authMiddleware)
+
+	r := gofight.New()
+
+	r.POST("/logout").
+		Run(handler, func(r gofight.HTTPResponse, rq gofight.HTTPRequest) {
+			assert.Equal(t, http.StatusOK, r.Code)
+			assert.Equal(t, fmt.Sprintf("%s=; Path=/; Domain=%s; Max-Age=0", cookieName, cookieDomain), r.HeaderMap.Get("Set-Cookie"))
```

Comment:
@appleboy I see you're also the gofight maintainer, what would you recommend here? It looks like I can't call `r.Header()` since `gofight.HTTPResponse` is an alias of `ResponseRecorder`

Reply:
@cmwright I will update after merging the PR